components/automate-ui/src/app/page-components/telemetry-checkbox/telemetry-checkbox.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
You need not to create telemetry-checkbox.module. ts file. You can import telemetry-checkbox component user modules.

Reply:
Same component not allowed to add in multiple modules, so I created separate module for this component.